### DISCUSSION TITLE
RHF package.json - enable ES6 module imports on server

### DISCUSSION THREAD

**Discussion by karanaditya993** [react-hook-form/react-hook-form#7033] [General] [answered]
@bluebill1049 Hi RHF team! Wondering if there is any harm in adding the `type: "module"` property to the `package.json` file for RHF. I am trying to import RHF in a node server and run into issues with the bundle using ES6 imports. Adding `type: "module"` to 
the `package.json` would solve the issues. Thanks

Error:
```
import { Controller, useFormContext } from "react-hook-form";
         ^^^^^^^^^^
SyntaxError: Named export 'Controller' not found. The requested module 'react-hook-form' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

**Comment by bluebill1049** [comment, marked as answer]
Sounds good. Would u like to send us a PR?

**Reply by karanaditya993** [reply]
Will do, thanks @bluebill1049

**Reply by karanaditya993** [reply]
@bluebill1049 do I need write access to the repo? Wondering if there are PR/contribution guidelines to the RHF repo. Thanks

**Reply by bluebill1049** [reply]
you can fork the repo and send us a PR 👍

**Reply by karanaditya993** [reply]
Ah, it looks like I need to fork and make a PR into upstream

**Reply by karanaditya993** [reply]
@bluebill1049 at your earliest convenience. Thanks again

https://github.com/react-hook-form/react-hook-form/pull/7034
